PromiseTests/delay.swift

```diff
@@ -20,6 +20,9 @@ struct SimpleError: Error, Equatable {
     
 }
 
+struct AlternativeError: Error, Equatable {
+
+}
```

Comment:
I needed a second error type besides `SimpleError`. I figured this would be a logical place to put it, even though it's only used by one test. Let me know if you have a better name or more reasonable place to put this!